scripts/eslint/es6Config.js

```diff
@@ -0,0 +1,18 @@
+const eslintrc = require('../../.eslintrc');
+
+const ERROR = 2;
+
+module.exports = Object.assign({}, eslintrc, {
+  parser: 'espree',
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+      experimentalObjectRestSpread: true,
```

Comment:
Why do we need these features for ES6? I’d expect both to be false.

Reply:
Cause this files [scripts/release/build-commands/parse-build-parameters.js](https://github.com/facebook/react/blob/master/scripts/release/build-commands/parse-build-parameters.js#L54) use this syntax.
So we don't allow this syntax?

Reply:
Why does it work? Did it get enabled by default in some Node version?

In this case let’s leave it on. But JSX shouldn’t be.

Reply:
I think that's because our `devEngines` is node `v8.x`, so this syntax works.